components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -475,12 +484,34 @@ - (void)viewDidDisappear:(BOOL)animated {
 
 - (void)preferredContentSizeDidChangeForChildContentContainer:(id<UIContentContainer>)container {
   [super preferredContentSizeDidChangeForChildContentContainer:container];
+  if ([container isKindOfClass:[UIViewController class]]) {
+    UIViewController *containerViewController = (UIViewController *)container;
+    if (containerViewController == self.contentViewController) {
+      CGFloat height = containerViewController.preferredContentSize.height;
+      [self resetInitialDrawerFactorWithHeight:height];
+    }
+  }
   _contentHeaderTopInset = NSNotFound;
   _contentHeightSurplus = NSNotFound;
   _addedContentHeight = NSNotFound;
   [self.view setNeedsLayout];
 }
 
+- (void)resetInitialDrawerFactorWithHeight:(CGFloat)height {
+  if (_contentVCPreferredContentSizeHeightCached == 0) {
```

Comment:
Tip: Drive-by comment (law this in an email).  Please use `MDCCGFloatEqual(_contentVCPreferredContentSizeHeightCached, 0)` to account for rounding errors, etc.